gelatio_boy_control/launch/scooping_planning.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+
+  <node name="scooping_planning" pkg="gelataio_boy_control" type="scooping_planning" respawn="false" output="screen">
```

Comment:
Maybe we should add the dependency to the move_it node. Right now we have to start scooping_planning and move_it separately.